disperser/dataapi/ejector.go

```diff
@@ -178,8 +178,11 @@ func (e *Ejector) Eject(ctx context.Context, nonsigningRate *OperatorsNonsigning
 	e.metrics.UpdateEjectionGasUsed(receipt.GasUsed)
 
 	// TODO: get the txn response and update the metrics.
+	ejectionResponse := &EjectionResponse{
+		TransactionHash: txn.Hash().Hex(),
```

Comment:
I'd take the transaction hash from `receipt`. 
The transaction hash can change when it gets broadcasted by Fireblocks.

Reply:
Good point